.config/nextest.toml

```diff
@@ -0,0 +1,2 @@
+[profile.default.junit]
+path = "junit.xml"
```

Comment:
junit??????

Reply:
yeah it's just an export format, so that test failure reports can (hopefully) be uploaded as artifacts and viewed from the github checks UI for the offending PR

https://nexte.st/book/junit.html